deploy/crds/jaegertracing_v1_jaeger_crd.yaml

```diff
@@ -1,3 +1,6 @@
+# ATTENTION: this file is kept here for backwards compatibility reasons
+# use the new location: jaegertracing.io_jaegers_crd.yaml . If there are
+# inconsistencies between the files, this one is considered wrong.
```

Comment:
+1 it would be cool to print deprecation message when it is used. Not sure if that is possible though.